Core/Core.csproj

```diff
@@ -5,13 +5,13 @@
     <RootNamespace>JetBrains.SymbolStorage</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="AWSSDK.CloudFront" Version="3.7.103.5" />
-    <PackageReference Include="AWSSDK.S3" Version="3.7.101.51" />
+    <PackageReference Include="AWSSDK.CloudFront" Version="4.0.0.5" />
+    <PackageReference Include="AWSSDK.S3" Version="4.0.0.7" />
     <PackageReference Include="JetBrains.Annotations" Version="2024.3.0" />
     <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.1.1" />
-    <PackageReference Include="Microsoft.SymbolStore" Version="1.0.626901" />
+    <PackageReference Include="Microsoft.SymbolStore" Version="1.0.627602" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageReference Include="System.Linq.Async" Version="6.0.1" />
-    <PackageReference Include="MSFTCompressionCab" Version="1.0.0" />
+    <PackageReference Include="WixToolset.Dtf.Compression.Cab" Version="6.0.0" />

```

Comment:
Is it acceptable to use the package with their license?
https://www.nuget.org/packages/WixToolset.Dtf.Compression.Cab/6.0.0/License